tests/sca_package/test_scanner.py

```diff
@@ -1,60 +1,50 @@
 import json
+import os
 from pathlib import Path
 
 import pytest
-from mock import AsyncMock, MagicMock
+from mock import AsyncMock
 from pytest_mock import MockerFixture
 
-from checkov.sca_package.scanner import Scanner
+from checkov.sca_package.scanner import Scanner, CHECKOV_SEC_IN_WEEK
 
 
-def test_setup_twistcli_exists(mocker: MockerFixture, tmp_path: Path):
+def test_should_download_new_twistcli(tmp_path: Path):
     # given
     scanner = Scanner()
-
-    integration_mock = MagicMock()
-    mocker.patch(
-        "checkov.common.bridgecrew.vulnerability_scanning.integrations.package_scanning.package_scanning_integration.download_twistcli",
-        side_effect=integration_mock,
-    )
-
-    # prepare local paths
     twistcli_path = tmp_path / "twistcli"
-    twistcli_path.touch()
     scanner.twistcli_path = twistcli_path
 
-    # when
-    scanner.setup_twictcli()
-
     # then
-    assert twistcli_path.exists()
-    integration_mock.assert_not_called()
+    assert scanner.should_download()
 
 
-def test_setup_twistcli_not_exists(mocker: MockerFixture, tmp_path: Path):
+def test_not_should_download_twistcli(tmp_path: Path):
     # given
     scanner = Scanner()
+    os.environ["CHECKOV_EXPIRATION_TIME_IN_SEC"] = str(CHECKOV_SEC_IN_WEEK)
+    twistcli_path = tmp_path / "twistcli"
+    twistcli_path.touch()
+    scanner.twistcli_path = twistcli_path
 
-    def download_twistcli(cli_file_name: Path):
-        cli_file_name.touch()
+    # then
+    assert not scanner.should_download()
+    # cleaning mock file
+    twistcli_path.unlink()
```

Comment:
no need to clean up, since you are using `tmp_path` from `pytest`, it will be cleaned up automatically after each test.